src/app/shared/RelayCard.svelte

```diff
@@ -174,4 +180,27 @@
       {/if}
     </div>
   {/if}
-</div>
+</AltColor>
+
+{#if details}
+  <Modal mini onEscape={() => (details = false)} class="md:hidden">
```

Comment:
Also, [nitpick], I usually set up named click handlers because inline ones break the syntax highlighting in my editor 😶. `boolCtrl` is a helper function I made for this that I don't know if I like yet.

Reply:
Could not use boolCtrl this time because `details` is binded to the relayCardActions child component
Also for some reason escape is triggered twice on close, toggling twice leaves the modal open 😰